relayer/relays/beacon/header/syncer/util/util.go

```diff
@@ -48,6 +48,18 @@ func ToUint64(stringVal string) (uint64, error) {
 	return intVal, err
 }
 
+func ToUint64AllowEmpty(stringVal string) (uint64, error) {
```

Comment:
Is this used somewhere?

Reply:
Yeah, used [here](https://github.com/Snowfork/snowbridge/blob/61dadc99ac650d5c424c033121df7169effd6f61/relayer/relays/beacon/header/syncer/api/api_response.go#L652) for backward compatible.